packages/eventual-send/test/test-deep-stacks.js

```diff
@@ -1,4 +1,4 @@
-// This file does not start with "test-" because it is useful as an
+// This file is not very useful as an
 // automated test. Rather, its purpose is just to run it to see what a
 // deep stack looks like.
```

Comment:
This seems to suggest that these (and similar tests in ses/errors) are really better framed as demos.